### ISSUE TITLE
SRM tokens stuck in bridge from solana wallet

### ISSUE THREAD

**Issue by matoshie** [solana-labs/oyster#263]
I transferred erc20 SRM tokens to my solana wallet through the bridge and when they showed up as an unknown token in my wallet, I tried to transfer them back.
Not sure what happened, but they aren't in my solana wallet and they never got transferred back to my erc20 wallet.

TX ID
https://explorer.solana.com/tx/4HHbuCXC1Z75EwZUnpoByt2S9RtQQR3V5RyGDdu386mNjfnqaqQ8qmqC816wDCM4udbyhj1Up8qun4oG67LGc51x

**Comment by leoluk** [comment]
The tx you provided failed with `Error: insufficient funds`, no tokens were transferred.

**Comment by matoshie** [comment]
here are the TX's that have happened with this token.
https://explorer.solana.com/address/9fAHc2vxEe3kAzrXJbvCHHqHcJn2yWYkrzXwkqLUhQzk

I don't know why it showed that I had insufficient funds, all I did was 2 transfers. One from the SRM erc20 tokens to my solana wallet which happened fine, but showed up as an unknown token and then transferred them back. When I transferred them back, they were gone from my solana wallet an didn't show up in my erc20 wallet.

Here's the last successful transaction ID before the failed one
https://explorer.solana.com/tx/5eyXqDkTFX5p4CrobhKnR5s23mNuruYNPcDsdXMkXQfb6zJK8x9PFdkJkfmF3F9TzNGoG2NaHhcZYWys1FDfSgFU
